logdna.gemspec

```diff
@@ -13,9 +12,7 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/logdna/ruby'
   spec.license       = 'MIT'
 
-  spec.files         = `git ls-files -z`.split("\x0").reject do |f|
-    f.match(%r{^(test|spec|features)/})
-  end
+  spec.files         = Dir.glob("{lib}/**/*") + %w(LICENSE.txt README.md)
```

Comment:
The idea behind the PR is reasonable but isn't this too broad?

Reply:
I'll pull out the #11 changes from here. No, I don't thin the files selection is broad. It includes only the functional files. And I included the .md and .txt files because this is what people seems to do for ruby gems. Although, I don't see why they include them.